nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -92,12 +92,18 @@ let
       ] else []
     ) env))));
 
-  mastodonEnv = pkgs.writeShellScriptBin "mastodon-env" ''
+  mastodonTootctl = pkgs.writeShellScriptBin "mastodon-tootctl" ''
```

Comment:
It would be nice at some point to have a generic trivial builder for such scripts.

Reply:
Not sure how easy that would be since so far every wrapper, mastodon-tootctl, nextcloud-occ and healthchecks-manage is very different except for the sudo part.

Reply:
Gotcha